components/tile/_tile.scss

```diff
@@ -57,11 +61,11 @@
 	}
 
 	.sdc-tile-header {
-		height: $body-font-1;
-		line-height: $header-height;
+		height: $heading-font-5 + 2px;
 		@include ellipsis($display: block);
-
-		font-size: $body-font-1;
+		font-size: $heading-font-5;
+		margin-top: -4px;
```

Comment:
Remove margin-top and margin-left, the position of the text is in the top left defined by the padding of the tile.